add infrastructure folder for espace (#303)
Conflux-Chain/conflux-documentation: docs/espace/build/infrastructure/infrastructure.mdx

---
sidebar_position: 1
title: Infrastructures
displayed_sidebar: eSpaceSidebar
---

> Web3 dApps's development is a complex process that requires a lot of different tools and services. Currently, there are a lot of different solutions on eSpace that can be used to build a dApp:

Reviewer: ```suggestion
Developing Web3 dApps is a multifaceted process, necessitating a variety of tools and services. Presently, numerous solutions are available on eSpace for constructing a dApp:
```

> - [RPC Service Providers](./RPC-Provider): Enterpise-grade RPC service providers that provide access to the blockchain network.

Reviewer: ```suggestion
- [RPC Service Providers](./RPC-Provider): These are enterprise-grade providers offering access to the blockchain network.
```

> - [TheGraph](./graph): TheGraph is a decentralized protocol for indexing and querying data from blockchains.

Reviewer: ```suggestion
- [TheGraph](./graph): TheGraph is a decentralized protocol designed for indexing and querying blockchain data.
```

> - [Oracles](./oracles): Oracles are data feeds that make off-chain data sources available to the blockchain for smart contracts.

Reviewer: ```suggestion
- [Oracles](./oracles): Oracles provide a bridge for off-chain data sources to be accessible within the blockchain, essential for smart contracts.
```
